routes/article.js

```diff
@@ -13,24 +13,24 @@ router.post('/vote', (req, res) => {
     if (amount === 0) {
         res.status(500).send({
             err: "Invalid value for vote",
-            reqBody: req.body
+            givens: req.body
```

Comment:
Let's also be open to other words that could describe the property just as well (I can't think of a better one rn though)